### DISCUSSION TITLE
DUDA BACK

### DISCUSSION THREAD

**Discussion by FernandoBarajasValdivia** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#578] [Q&A]
Les llego su correo para entrar al repo de back? a mi no :(
![image](https://user-images.githubusercontent.com/99165285/161367667-af69710b-dec9-4ceb-b302-73b9c105a4bf.png)

**Comment by carlos-gmx3** [comment]
Checa si puedes ver tu equipo en la sección Teams de la Organización
https://github.com/orgs/LaunchX-InnovaccionVirtual/teams

**Reply by ismagician** [reply]
A mi no me llegó correo pero ya vi que me agregó al equipo, gracias 👍

**Comment by Nukario** [comment]
Tampoco lo he recibido, en teams dijeron que apagaron los servidores y seguiran hoy apartir de las 10am el registro que si ya nos registramos esperemos a esa hora para que nos llegue el correo que no es necesario registrarnos otra vez

**Comment by ErickRueda** [comment]
Estoy igual, ya intente un par de veces y nada xC
